code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -372,7 +372,22 @@
 				round_statistics.current_map.total_marine_victories++
 				round_statistics.current_map.total_marine_majors++
 		if(MODE_INFESTATION_X_MINOR)
-			musical_track = pick('sound/theme/neutral_melancholy1.ogg','sound/theme/neutral_melancholy2.ogg')
+			var/living_player_list[] = count_humans_and_xenos(EvacuationAuthority.get_affected_zlevels())
+			if(living_player_list[1] && !living_player_list[2]) // If Xeno Minor but Xenos are dead and Humans are alive, see which faction is the last standing
+				var/headcount = count_per_faction()
+				var/majority = 0.5 // What percent do we consider a 'majority?'
+				if ((headcount[2] / headcount[5]) > majority) // headcount[2] = Weyland-Yutani
+					musical_track = pick('sound/theme/LastManStanding_WY.ogg')
+				else if ((headcount[3] / headcount[5]) > majority) // headcount[3] = UPP
+					musical_track = pick('sound/theme/LastManStanding_UPP.ogg')
+				else if ((headcount[4] / headcount[5]) > majority) // headcount[4] = CLF
+					musical_track = pick('sound/theme/LastManStanding_CLF.ogg')
+				else if ((headcount[1] / headcount[5]) > majority) // headcount[1] = Marines
```

Comment:
```suggestion
				var/headcount = count_per_faction()
				var/living= headcount[5]
				var/majority = 0.5 // What percent do we consider a 'majority?'
				if ((headcount[2] / living) > majority) // headcount[2] = Weyland-Yutani
					musical_track = pick('sound/theme/LastManStanding_WY.ogg')
				else if ((headcount[3] / living) > majority) // headcount[3] = UPP
					musical_track = pick('sound/theme/LastManStanding_UPP.ogg')
				else if ((headcount[4] / living) > majority) // headcount[4] = CLF
					musical_track = pick('sound/theme/LastManStanding_CLF.ogg')
				else if ((headcount[1] / living) > majority) // headcount[1] = Marines
```

Instead of headcount[5] why not make it easier for someone in the future by assigning it to a variable, since you use it often.

Reply:
Also a good point, in my mind I originally thought the array might have more values added and the total would be the final entry in a "clean" way.  Honestly it doesnt matter and I think this reads much nicer without having to understand both parts. Another good suggestion.